UnitTests.NetCore/Moq.NetCore.Tests/project.json

```diff
@@ -0,0 +1,62 @@
+{
+  "version": "1.0.0-*",
+  "description": "Moq.NetCore.UnitTests Class Library",
+  "authors": [ "" ],
+  "packOptions": {
+    "tags": [ "" ],
+    "projectUrl": "",
+    "licenseUrl": ""
+  },
+
+  "buildOptions": {
+    "keyFile": "../../Moq.snk",
+    "compile": {
+      "include": [
+        "../../UnitTests/**/*.cs"
+      ],
+      "exclude": [
+        "../../UnitTests/Library/**/*.cs",
+        "../../UnitTests/Pex/**/*.cs",
+        "../../UnitTests/Properties/**/*.cs"
+      ]
+    }
+  },
+
+  "dependencies": {
+    "xunit": "2.1.0"
+  },
+
+  "testRunner": "xunit",
+  "frameworks": {
+    "netcoreapp1.0": {
+      "buildOptions": {
+        "define": [ "NETCORE" ]
+      },
+      "dependencies": {
+        "Castle.Core": "4.0.0-alpha001",
+        "ClassLibrary1": {
+          "target": "project",
+          "version": "1.0.0-*"
+        },
+        "ClassLibrary2": {
+          "target": "project",
+          "version": "2.0.0-*"
+        },
+        "Moq": {
+          "target": "project",
+          "version": "99.99.99-wrapped"
+        },
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0"
+        },
+        "System.Collections.Specialized": "4.0.1",
+        "System.ComponentModel.Primitives": "4.1.0",
+        "System.Diagnostics.TraceSource": "4.0.0",
```

Comment:
Should this be a dependency in `source.netcore/project.json` instead to avoid manually adding it in each project?
See also a related discussion: http://stackoverflow.com/questions/37288385/moq-netcore-failing-for-net-core-rc2